x-pack/plugins/security_solution/public/explore/hosts/pages/navigation/host_risk_score_tab_body.tsx

```diff
@@ -92,8 +91,8 @@ export const HostRiskScoreQueryTabBody = ({
     isDeprecated: isDeprecated && !loading,
   };
 
-  if (RiskScoreUpsell) {
-    return <RiskScoreUpsell />;
+  if (!isAuthorized) {
+    return null;
```

Comment:
I left the tab there so it is easier for those who add the Upsell component. And it also feels like a waste to add extra code for hiding the tab that will be deleted very soon. But I understand your point. It would be more consistent while we don't have the Upsell component. As a compromise, I added a TODO message. 
<img width="1788" alt="Screenshot 2023-07-26 at 10 22 04" src="https://github.com/elastic/kibana/assets/1490444/245a831a-4eee-40d9-90a4-66a75c997472">